sdk/storage/Azure.Storage.Common.DataMovement/src/FilesystemScanner.cs

```diff
@@ -5,108 +5,142 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+using System.Linq;
 
 namespace Azure.Storage.Common.DataMovement
 {
     /// <summary>
     /// FilesystemScanner class.
     /// </summary>
-    public static class FilesystemScanner
+    public class FilesystemScanner
     {
         /// <summary>
-        /// Enumerates all files pointed to by the provided path, including those in subdirectories (if path is a directory).
+        /// Fully qualified path(s) equivalent to user-supplied path(s).
         /// </summary>
-        /// <param name="path">Filesystem location.</param>
-        /// <returns>Enumerable list of absolute paths containing all relevant files the user has permission to access.</returns>
-        public static IEnumerable<string> ScanLocation(string path)
+        public IEnumerable<string> FullPaths
```

Comment:
should this be private?

Reply:
I saw some similar track 1 classes had paths public, and followed suit; although, there isn't anything useful added here by making them public. I'll change this to private.

Reply:
yeah, if there's no reason to make it public then we should make it private. We shouldn't add functionality without need.